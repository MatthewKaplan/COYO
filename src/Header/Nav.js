import React from 'react';
import '../Styles/Nav.scss';
import logo from './logo.png'
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'


export default function Nav(props) {
  return (
    <nav className="navBar">
      <div className="logo">
        <a href="#home" className="logoLink"> <img src={logo} alt="CoYo Logo" className="logoImg"/></a>
      </div>
      <ul className="navList">
        <li className="facebook socialMedia">
         <a href="https://www.facebook.com">
          <FontAwesomeIcon icon={['fab', 'facebook-square']} />
         </a>
        </li>
        <li className="twitter socialMedia">
         <a href="www.twitter.com">
          <FontAwesomeIcon icon={['fab', 'twitter']} />
         </a>
        </li>
        <li className="instagram socialMedia">
         <a href="www.instagram.com">
          <FontAwesomeIcon icon={['fab', 'instagram']} />
         </a>
        </li>
      </ul>
    </nav>
  )
}