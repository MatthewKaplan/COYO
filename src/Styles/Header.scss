@import 'Mixins';

.banner {
  width: 100%;
  background: no-repeat bottom url("./colorado-mountains.jpg");
  filter: brightness(80%);
  color: $banner-color;
  text-align: center;
  padding: 100px;
}

.co,
.yo {
  font-family: $primary-font;
  font-weight: 800;
  line-height: .8;
  font-size: 100px;
  text-shadow: $coyo-text-shadow1 3px 3px 0px, $coyo-text-shadow2 5px 5px 0px;
}

h1 {
  margin: 0 0 75px 0;
}

.fullNameHeading {
  font-family: $heading-font;
}