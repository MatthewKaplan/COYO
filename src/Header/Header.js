import React, { Component } from 'react';
import '../Styles/Header.scss';
import Nav from './Nav'

export default function Header(props) {
  return (
    <header className="pageHeader">
      <Nav />
      <section className="banner">
        <h1 className="coyoHeading">
          <div className="headingText co">CO</div>
          <div className="headingText yo">YO</div>
        </h1>
        <h2 className="fullNameHeading">Colorado Yoga Connection</h2>
      </section>
    </header>
  )
}  