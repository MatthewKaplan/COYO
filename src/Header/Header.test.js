import React from 'react';
import Header from './Header.js';
import { shallow } from 'enzyme';


describe('Header', () => {

})