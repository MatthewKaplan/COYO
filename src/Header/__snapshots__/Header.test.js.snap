// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <section
          className="banner"
        >
          <h1
            className="coyoHeading"
          >
            <div
              className="headingText co"
            >
              CO
            </div>
            <div
              className="headingText yo"
            >
              YO
            </div>
          </h1>
          <h2
            className="fullNameHeading"
          >
            colorado yoga collective
          </h2>
        </section>,
        <section
          className="temp"
        />,
      ],
      "className": "pageHeader",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1
              className="coyoHeading"
            >
              <div
                className="headingText co"
              >
                CO
              </div>
              <div
                className="headingText yo"
              >
                YO
              </div>
            </h1>,
            <h2
              className="fullNameHeading"
            >
              colorado yoga collective
            </h2>,
          ],
          "className": "banner",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="headingText co"
                >
                  CO
                </div>,
                <div
                  className="headingText yo"
                >
                  YO
                </div>,
              ],
              "className": "coyoHeading",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "CO",
                  "className": "headingText co",
                },
                "ref": null,
                "rendered": "CO",
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "YO",
                  "className": "headingText yo",
                },
                "ref": null,
                "rendered": "YO",
                "type": "div",
              },
            ],
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "colorado yoga collective",
              "className": "fullNameHeading",
            },
            "ref": null,
            "rendered": "colorado yoga collective",
            "type": "h2",
          },
        ],
        "type": "section",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "temp",
        },
        "ref": null,
        "rendered": null,
        "type": "section",
      },
    ],
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <section
            className="banner"
          >
            <h1
              className="coyoHeading"
            >
              <div
                className="headingText co"
              >
                CO
              </div>
              <div
                className="headingText yo"
              >
                YO
              </div>
            </h1>
            <h2
              className="fullNameHeading"
            >
              colorado yoga collective
            </h2>
          </section>,
          <section
            className="temp"
          />,
        ],
        "className": "pageHeader",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h1
                className="coyoHeading"
              >
                <div
                  className="headingText co"
                >
                  CO
                </div>
                <div
                  className="headingText yo"
                >
                  YO
                </div>
              </h1>,
              <h2
                className="fullNameHeading"
              >
                colorado yoga collective
              </h2>,
            ],
            "className": "banner",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <div
                    className="headingText co"
                  >
                    CO
                  </div>,
                  <div
                    className="headingText yo"
                  >
                    YO
                  </div>,
                ],
                "className": "coyoHeading",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "CO",
                    "className": "headingText co",
                  },
                  "ref": null,
                  "rendered": "CO",
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "YO",
                    "className": "headingText yo",
                  },
                  "ref": null,
                  "rendered": "YO",
                  "type": "div",
                },
              ],
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "colorado yoga collective",
                "className": "fullNameHeading",
              },
              "ref": null,
              "rendered": "colorado yoga collective",
              "type": "h2",
            },
          ],
          "type": "section",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "temp",
          },
          "ref": null,
          "rendered": null,
          "type": "section",
        },
      ],
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
